src/main/resources/i18n/messages.properties

```diff
@@ -119,10 +119,10 @@ FoldTab.64=<html><strong>Default:</strong> Enabled if filename prettifying is en
 FoldTab.65=Monitor played status of files
 FoldTab.66=<html><strong>Default:</strong> Disabled<br><strong>Notes:</strong> Folders must have "Monitor played status of files" enabled to have their contents included in the New Media folder.</html>
 FoldTab.67=Do nothing
-FoldTab.68=Add an overlay to the thumbnail
-FoldTab.69=Hide the file
-FoldTab.70=Move the file to a different folder
-FoldTab.71=Move the file to the recycle/trash bin
+FoldTab.68=Tag media
```

Comment:
@Nadahar "mark" seems good, although it seems to me that the text is a fallback rather than a feature on its own, since most renderers support thumbnails. It might make more sense to mention just the overlay here, and note the text fallback in the config and a tooltip.

If you think "media" is a better choice feel free to use it. I personally like "file" especially when talking about file operations like moving to folders and trash bins, but as usual I'm not insistent on it.

The other improvements in this PR look good to me.